pkg/messaging/mqtt/pubsub.go

```diff
@@ -21,13 +21,14 @@ const (
 )
 
 var (
-	errConnect                = errors.New("failed to connect to MQTT broker")
-	errSubscribeTimeout       = errors.New("failed to subscribe due to timeout reached")
-	errUnsubscribeTimeout     = errors.New("failed to unsubscribe due to timeout reached")
-	errUnsubscribeDeleteTopic = errors.New("failed to unsubscribe due to deletion of topic")
-	errNotSubscribed          = errors.New("not subscribed")
-	errEmptyTopic             = errors.New("empty topic")
-	errEmptyID                = errors.New("empty ID")
+	ErrConnect                = errors.New("failed to connect to MQTT broker")
+	ErrSubscribeTimeout       = errors.New("failed to subscribe due to timeout reached")
+	ErrUnsubscribeTimeout     = errors.New("failed to unsubscribe due to timeout reached")
```

Comment:
Is this a global variable or only used withing the package?

Reply:
They are used inside mqtt_test package, and may also be exported as required (The error variables are global in the nats and rabbitmq implementations. So, this is also to make mqtt uniform with them).